tests/sentry/web/api/tests.py

```diff
@@ -105,3 +105,27 @@ def test_superuser(self):
         assert data["user"]
         assert data["user"]["email"] == user.email
         assert data["user"]["isSuperuser"]
+
+    def test_url_prefix_unauthenticated(self):
+        resp = self.client.get(self.path)
+        assert resp.status_code == 200
+        assert resp["Content-Type"] == "application/json"
+
+        data = json.loads(resp.content)
+        assert not data["isAuthenticated"]
+        assert data["user"] is None
+        assert data["apiUrl"] is None
```

Comment:
We'll likely need an `apiUrl` when there is no authentication so that users can make login requests.